### DISCUSSION TITLE
打开V2版本的软件直接进如图界面，没有选择游戏目录这一步。

### DISCUSSION THREAD

**Discussion by Caponia** [cangzhang/ChampR#276] [General]
![image](https://github.com/cangzhang/champ-r/assets/33995465/aadbb9d0-1374-497b-83b8-2967f00dbdb8)
![image](https://github.com/cangzhang/champ-r/assets/33995465/7c0ef58c-c551-4ff6-b77b-9a57140c1ff6)

**Comment by cangzhang** [comment]
保持 LoL 运行即可，不需要选择目录，之后会增加提示

**Reply by Caponia** [reply]
但下载的出装配置文件没有直接下载到游戏内的相关目录中，不知道是什么原因。

**Reply by cangzhang** [reply]
是国服还是别的服务器？

**Reply by Caponia** [reply]
马服

**Reply by cangzhang** [reply]
勾选数据源了吗？没有勾选的话自然是没有导入的

**Reply by Caponia** [reply]
勾选数据源了，只是火箭上方显示的是Not connect to league of legend

**Reply by cangzhang** [reply]
录段使用 v2.0.2-b3 版本的视频看看？看一下终端窗口里是怎么输出的

**Comment by cangzhang** [comment]
https://t.me/champ_r/98

> 各位如果遇到连接不到客户端的问题，请检查自己的操作系统是否是 win10，以及 lol 安装路径是否包含中文。
> 
> 如果是 win10 且包含中文的话，将安装路径改成英文则可解决该问题。
> 
> 如果是 win11，中文路径应该没有问题（本机测试）。
> 
> 感谢 sfzhang 的帮助调查👍，如果有其他问题请各位继续反馈

**Comment by cangzhang** [comment]
https://github.com/cangzhang/champ-r/releases/tag/v2.0.2-b5
